api/src/utils/media/media-utils.ts

```diff
@@ -114,3 +118,59 @@ export const isZipMimetype = (mimetype: string): boolean => {
     regex.test(mimetype)
   );
 };
+
+export const parseUnknownMediaToDwCAArchive = (unknownMedia: UnknownMedia): DWCArchive => {
```

Comment:
Can we get JS doc on this explaining what it's for?

Reply:
I think I was trying reorganize some code but this ended up being useless. It has been removed